content/docs/reference-glossary.md

```diff
@@ -120,47 +125,49 @@ class Welcome extends React.Component {
 
 ### [`state`](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) {#state}
 
-A component needs `state` when some data associated with it changes over time. For example, a `Checkbox` component might need `isChecked` in its state, and a `NewsFeed` component might want to keep track of `fetchedPosts` in its state.
+קומפוננטה צריכה `state` כאשר נתונים כלשהם המשויכים אליה משתנים עם הזמן. לדוגמה, קומפוננטת `Checkbox` עשויה להזדקק ל-`isChecked` ב-`state` שלה, וקומפוננטת `NewsFeed` עשויה לרצות לעקוב אחרי `fetchedPosts` ב-`state` שלה.
 
-The most important difference between `state` and `props` is that `props` are passed from a parent component, but `state` is managed by the component itself. A component cannot change its `props`, but it can change its `state`.
+ההבדל החשוב ביותר בין `state` ל-`props` הוא ש-`props` מועברים מקומפוננטת הורה, אבל `state` מנוהל על ידי הקומפוננטה עצמה. קומפוננטה לא יכולה לשנות את ה-`props` שלה, אבל היא כן יכולה לשנות את ה-`state` שלה.
 
-For each particular piece of changing data, there should be just one component that "owns" it in its state. Don't try to synchronize states of two different components. Instead, [lift it up](/docs/lifting-state-up.html) to their closest shared ancestor, and pass it down as props to both of them.
+עבור כל פיסה מסוימת של נתונים משתנים, צריך להיות רק קומפוננטה אחת שתהיה "הבעלים" של אותו מידע ב-`state` שלה. אל תנסו לסנכרן `state` של שני קומפוננטות שונות. במקום זאת, [הרימו אותו למעלה](/docs/lifting-state-up.html) אל האב הקדמון המשותף הקרוב ביותר שלהם, והעבירו אותו למטה בתור `props` לשתי הקומפוננטות.
 
-## [Lifecycle Methods](/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class) {#lifecycle-methods}
+## [מתודות מעגל חיים](/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class) {#lifecycle-methods}
```

Comment:
Even though lifecycle is מעגל חיים, do you think it would also be correct to name it מתודות מחזוריות or מתודות בעלות מחזור חיים? Just a suggestion

Reply:
It relates to the methods of the lifecycle, it does not relate to methods that have lifecycle so "מתודות בעלות מחזור חיים" is not a good fit, and neither מתודות מחזוריות doesn't work I think since this option tends more towards the wrong option.
The methods are related to the lifecycle and not the other way around, so I think leaving it as is would be best.